tests/test_hd44780.py

```diff
@@ -0,0 +1,187 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+# Copyright (c) 2020 Richard Hull and contributors
+# See LICENSE.rst for details.
+
+"""
+Tests for the :py:class:`luma.lcd.device.hd44780` device.
+"""
+
+from luma.lcd.device import hd44780
+from luma.core.render import canvas
+from luma.core.util import bytes_to_nibbles
+
+from PIL import Image, ImageDraw
+from unittest.mock import Mock, call
+
+CLEAR = 0x01
```

Comment:
can these constants be moved to a temporary test class/enum and ref them like that.

Reply:
They were unnecessary.  I referenced the version of the constants from luma/lcd/const.py